.jenkins/pytorch/build.sh

```diff
@@ -293,6 +293,8 @@ else
     pushd ../cpp-build/caffe2
     WERROR=1 VERBOSE=1 DEBUG=1 python "$BUILD_LIBTORCH_PY"
     popd
+    echo "Building functorch"
+    install_functorch
```

Comment:
We can delete https://github.com/pytorch/pytorch/blob/bbe8d019f280478dc3b143f6988e3e5668499f28/.jenkins/pytorch/test.sh#L679 after this, right?

Reply:
I think you're right - I'll delete it and see if there's any fallout